tests/suites/smoke/build.sh

```
@@ -1,11 +1,5 @@
 run_build() {
-    OUT=$(make go-build 2>&1 || true)
-    if [ -n "${OUT}" ]; then
-        echo ""
-        echo "$(red 'Found some issues:')"
-        echo "\\n${OUT}"
-        exit 1
-    fi
+    make go-build 2>&1
```

Rather than be fancy, just let it fail!

---

if you still want to be fancy:

```
OUT=$(make go-build 2>&1 || echo "BUILD FAILED")
if [ "$OUT" =~ (BUILD FAILED) ]; then
  ....
fi
```

---

I'll think about it